### ISSUE TITLE
Unexpected renderer encountered.

### ISSUE THREAD

**Issue by nxxte** [pytube/pytube#1439]
so i was testing z program that i created using pytube everything worked perfectly until i searched this term "central cee let her go"
it actually worked fine and i downloaded the video but i got this output:
Unexpected renderer encountered.
Renderer name: dict_keys(['reelShelfRenderer'])
Search term: central cee let her go
Please open an issue at https://github.com/pytube/pytube/issues and provide this log output.
other than this output everything worked fine
pytube version 12.0.0

**Comment by iRingil** [comment]
I confirm, from time to time such an error also occurs:

> WARNING  search.py:153 [2022-12-16 16:00:03,959] - pytube.contrib.search - Unexpected renderer encountered.
> WARNING  search.py:154 [2022-12-16 16:00:03,975] - pytube.contrib.search - Renderer name: dict_keys(['reelShelfRenderer'])
> WARNING  search.py:155 [2022-12-16 16:00:03,975] - pytube.contrib.search - Search term: قمر ومنور
> WARNING  search.py:156 [2022-12-16 16:00:03,975] - pytube.contrib.search - Please open an issue at https://github.com/pytube/pytube/issues and provide this log output.

**Comment by ToniBossOfficial** [comment]
the same error with me!

**Comment by Pyp-3** [comment]
Same here.Any way it can be solved?

**Comment by mypartnadav** [comment]
also having the same issue

**Comment by Breaderick** [comment]
also having the same issue. tried both with version 12.1.0 and latest version. from what I gathered it might have something to do with YouTube shorts and the new YouTube redesign

**Comment by rgryta** [comment]
Yeah. ReelShelfRenderer are youtube shorts results when you're searching for a title.
I have fixed that issue back in June, but it has not been merged and it seems that youtube screwed with it a little as well since then. I have reapplied a fix and it seems to be working. You can install that fix by installing pytube: `pip3 install git+https://github.com/rgryta/pytube.git@main` until it's merged.
